tests/test_storage_manager_edge_cases.py

```
@@ -0,0 +1,301 @@
+        result = self.git_handler.commit("")
+
+        # Assert
+        # 空のメッセージでもデフォルトメッセージが使用される
+        self.assertTrue(result.success)
+        mock_run.assert_called()
+        call_args = mock_run.call_args[0][0]
+        self.assertIn("commit", call_args)
+
+    def test_add_files_with_glob_patterns(self):
+        """グロブパターンでのファイル追加をテスト"""
+        # テストをスキップ（GitHandlerはadd_filesメソッドを持たない）
```

Ruff (0.14.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)